spec/requests/logging_spec.rb

```diff
@@ -45,7 +45,7 @@
       expect(log_io.string).to include(
         'Parameters:     {"action"=>"create", "controller"=>"api/services", "format"=>"json", ' \
         '"body"=>{"action"=>"create", "resource"=>{"name"=>"new_service_1", ' \
-        '"options"=>{"password"=>"[FILTERED]"}}}}'
+        '"options"=>{"password"=>"[FILTERED]"}}},'
```

Comment:
This change and the one below is to avoid having to change these unrelated tests for the `request_id` change.  We're basically saying "check the string up to this point..." instead of the whole string, which will now include the request_id.